iceoryx_posh/test/moduletests/test_roudi_iceoyx_roudi_memory_manager.cpp

```diff
@@ -94,6 +103,17 @@ TEST_F(IceoryxRoudiMemoryManager_test, AcquiringIntrospectionMemoryManagerAfterC
     EXPECT_THAT(result, Not(Eq(iox::nullopt_t())));
 }
 
+TEST_F(IceoryxRoudiMemoryManager_test, AcquiringDiscoveryMemoryManagerAfterCreateAndAnnounceMemoryIsSuccessful)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "ce42929a-0721-41b9-bf08-b48e53fe901e");
+    auto tr = m_roudiMemoryManagerTest->createAndAnnounceMemory();
```

Comment:
What does `tr` stand for?

Reply:
I guess short for `test result`. I just c&p a test and adjusted it.

Reply:
Forget my question ;)